src/Sarif.Converters/ContrastSecurityConverter.cs

```diff
@@ -429,13 +440,10 @@ private Result ConstructCrossSiteScriptingResult(ContrastLogReader.Context conte
             return result;
         }
 
-
                     // A cross-site scripting vulnerability was observed as untrusted data '{0}' on  '{1}' was accessed within  '{2}' control and observed going into the HTTP response without validation or encoding.
-    
 
```

Comment:
this comment looks out of place?

Reply:
i mention this since we are cleaning up the file and standardizing our code pattern! :)

---
In reply to: [272415479](https://github.com/Microsoft/sarif-sdk/pull/1401#discussion_r272415479) [](ancestors = 272415479)

Reply:
Thanks, fixed.

---
In reply to: [272415599](https://github.com/Microsoft/sarif-sdk/pull/1401#discussion_r272415599) [](ancestors = 272415599,272415479)